tuf/util.py

```diff
@@ -952,6 +991,120 @@ def load_json_file(filepath):
 
 
 
+
+
+def load_der_file(filepath):
+  """
+  <Purpose>
+    Read in a DER file and output a Python dictionary in standard TUF
+    format containing the translated contents of the DER file.
+
+  <Arguments>
+    filepath:
+      Absolute path of DER file.
+
+  <Exceptions>
+    tuf.FormatError: If 'filepath' is improperly formatted.
+
+    tuf.Error: If 'filepath' cannot be deserialized to a Python object.
+
+    IOError in case of runtime IO exceptions.
+
+  <Side Effects>
+    None.
+
+  <Return>
```

Comment:
Yes, but I'm not sure that's possible in the case of DER, unfortunately. You have to provide an ASN1 definition for the metadata in order to decode DER, and it's pretty rigid.